ultravox/data/dataset_config.py

```diff
@@ -1,16 +1,24 @@
 import dataclasses
+from enum import Enum
 from typing import List, Optional
 
 from pydantic import BaseModel
 
 
+class InterleaveStopStrategy(str, Enum):
```

Comment:
I think I would keep this in datasets.py as it's specific to the `InterleaveDataset` class there. Also can the enum names be capitalized or does that cause a problem for config serialization?